GoogleForCreators/web-stories-wp
# Update text-sets to use TT for all caps words

## Context

With the addition of uppercase toggle for text, text-sets needed to be updated to use TT.

## Summary

Updates all text sets JSON to add `text-transform` where needed.

## User-facing changes



## Testing Instructions

- Pick any text-set from the panel that has an all caps word.
- Select the text layer, the `TT` button should be highlighted.

## Checklist




---

Fixes #7658
